packages/dds/tree/src/test/utils.ts

```diff
@@ -525,17 +524,23 @@ export class SharedTreeTestFactory extends SharedTreeFactory {
 	}
 }
 
-export function noRepair(): DeltaProtoNode[] {
-	assert.fail("Unexpected request for repair data");
-}
-
 export function validateTree(tree: ITreeCheckout, expected: JsonableTree[]): void {
 	const actual = toJsonableTree(tree);
 	assert.deepEqual(actual, expected);
 }
 
 const schemaCodec = makeSchemaCodec({ jsonValidator: typeboxValidator });
 
+export function checkRemovedRootsAreSynchronized(trees: readonly ITreeCheckout[]) {
+	if (trees.length > 1) {
+		const baseline = nestedMapFromFlatList(trees[0].getRemovedRoots());
+		for (const tree of trees.slice(1)) {
```

Comment:
why slice here?

Reply:
Because there's no point comparing `trees[0]` with itself. Is there a better way to do this?